components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -542,6 +542,12 @@ subroutine micro_p3_init(pbuf2d)
    call addfld('P3_mtend_NUMICE',  (/ 'lev' /), 'A', 'kg/kg/s', 'P3 Tendency for ice cloud number due to micro processes')
    call addfld('P3_mtend_Q',       (/ 'lev' /), 'A', 'kg/kg/s', 'P3 Tendency for water vapor due to micro processes')
    call addfld('P3_mtend_TH',      (/ 'lev' /), 'A', 'kg/kg/s', 'P3 Tendency for potential temp. number due to micro processes')
+   ! phase change tendencies
+   call addfld('vap_liq_exchange',  (/ 'lev' /), 'A', 'kg/kg/s', 'Tendency for conversion from vapor phase to liquid phase')
+   call addfld('vap_ice_exchange',  (/ 'lev' /), 'A', 'kg/kg/s', 'Tendency for conversion from vapor phase to frozen phase')
+   call addfld('liq_ice_exchange',  (/ 'lev' /), 'A', 'kg/kg/s', 'Tendency for conversion from liquid phase to frozen phase')
+   call addfld('vap_cld_exchange',  (/ 'lev' /), 'A', 'kg/kg/s', 'Tendency for conversion from vapor phase to cloud category')
+
```

Comment:
Do we want the long name to use to/from rather than just from to indicate that we're also including liquid->vapor transitions (for example)?

Reply:
hmmm, yes I think you are right.  Maybe something like "from/to vapor phase to/from liquid phase" as a way to get the sign correct?